css/README.md

```diff
@@ -0,0 +1,45 @@
+# CSS Style Guide
+
+## General Guidelines for CSS Code
+
+### Readability
+
+- As with any code, ensure CSS code is optimized for human reading
+- Use descriptive class/variable names and consistent naming conventions
+- Use tools like *prettier* to maintain well-formatted CSS files
+- Leave comments in CSS code when necessary
+
+### Component Libraries
+
+- The usage of component libraries like *Ant Design* or *MUI* is encouraged
+- Integrate with the library's theme system to provide base design and components for your app
+
+### DRY
+
+- Do not copy-paste similar CSS or components
+- Extract common code into a new component where necessary
+- Use utility CSS classes where appropriate
+- Extract variables for reusing common colors, fonts, metrics, etc.
+
+### CSS Modularity
+
+- Avoid using global styles and selectors
```

Comment:
maybe the exception to this rule is "theming" but that should be mostly for colors etc (mostly in form of variables - either directly CSS variables or sass variables).

Reply:
Added